Mathlib/LinearAlgebra/TensorProduct/Basic.lean

```diff
@@ -1116,11 +1120,19 @@ def lTensor (f : N →ₗ[R] P) : M ⊗[R] N →ₗ[R] M ⊗[R] P :=
   TensorProduct.map id f
 #align linear_map.ltensor LinearMap.lTensor
 
-/-- `rTensor f M : N₁ ⊗ M →ₗ N₂ ⊗ M` is the natural linear map induced by `f : N₁ →ₗ N₂`. -/
+/-- `rTensor M f : N₁ ⊗ M →ₗ N₂ ⊗ M` is the natural linear map induced by `f : N₁ →ₗ N₂`. -/
 def rTensor (f : N →ₗ[R] P) : N ⊗[R] M →ₗ[R] P ⊗[R] M :=
   TensorProduct.map f id
 #align linear_map.rtensor LinearMap.rTensor
 
+theorem _root_.TensorProduct.coe_congr_left_refl (f : N ≃ₗ[R] P) :
+    (TensorProduct.congr (LinearEquiv.refl R M) f).toLinearMap = lTensor M f :=
+  rfl
+
+theorem _root_.TensorProduct.coe_congr_right_refl (f : N ≃ₗ[R] P) :
+    (TensorProduct.congr f (LinearEquiv.refl R M)).toLinearMap = rTensor M f :=
+  rfl
```

Comment:
I think simp normal forms are the following:
```suggestion
theorem _root_.TensorProduct.coe_congr_left_refl (f : N ≃ₗ[R] P) :
    TensorProduct.congr (.refl R M) f = lTensor M (f : N →ₗ[R] P) :=
  rfl

theorem _root_.TensorProduct.coe_congr_right_refl (f : N ≃ₗ[R] P) :
    TensorProduct.congr f (.refl R M) = rTensor M (f : N →ₗ[R] P) :=
  rfl
```
But maybe it's more appropriate to introduce LinearEquiv.lTensor/rTensor.